locale/fi/LC_MESSAGES/django.po

```diff
@@ -2427,66 +2531,107 @@ msgid "7 hours"
 msgstr "7 tuntia"
 
 #: reservation_units/enums.py
+msgctxt "ReservationKind"
+msgid "Direct"
+msgstr "Suora"
+
+#: reservation_units/enums.py
+msgctxt "ReservationKind"
+msgid "Season"
+msgstr "Kausi"
+
+#: reservation_units/enums.py
+msgctxt "ReservationKind"
+msgid "Direct and season"
+msgstr "Suora ja kausi"
+
+#: reservation_units/enums.py
+msgctxt "PricingType"
+msgid "Paid"
+msgstr "Maksettu"
```

Comment:
Looks like this should more properly be "Maksullinen" as well.

Reply:
Fixed